plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduIntegrationDateColumns.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu;
+
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.sql.TestTable;
+import io.trino.testing.sql.TrinoSqlExecutor;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.sql.Date;
+import java.time.LocalDate;
+
+import static io.trino.plugin.kudu.KuduQueryRunnerFactory.createKuduQueryRunner;
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestKuduIntegrationDateColumns extends AbstractTestQueryFramework
```

Comment:
Can we introduce type mapping test like we handle in other connectors.

Reply:
do ask to move all tests to type mapping or can i just start by this one ?

Reply:
It would be nice if we could have a type mapping test for all formats (incl. DATE as STRING) and then we could apply this change.

Reply:
waiting for type mapping report back in kudu